MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+    tileLayer.ZoomPanBehavior = zoomPanBehavior;
+    MapMarker mapMarker = new MapMarker();
+    mapMarker.Latitude = 38.909804;
+    mapMarker.Longitude = -77.043442;
+    MapMarkerCollection mapMarkers = new MapMarkerCollection();
+    mapMarkers.Add(mapMarker);
+    tileLayer.Markers = mapMarkers;
+    tileLayer.MarkerTemplate = CreateDataTemplate();
+    MapLatLngBounds bounds = new MapLatLngBounds();
+    bounds.Northeast = new MapLatLng(38.909804, -77.043442);
+    bounds.Southwest = new MapLatLng(38.909804, -77.043442);
+    tileLayer.MapLatLngBounds = bounds;
+    map.Layer = tileLayer;
+    this.Content = map;
+}
+
+private DataTemplate CreateDataTemplate()
+{
+    return new DataTemplate(() =>
+    {
+        var stackLayout = new StackLayout();
+        stackLayout.HorizontalOptions = LayoutOptions.StartAndExpand;
+        stackLayout.VerticalOptions = LayoutOptions.Center;
+        var image = new Image
+        {
+            Source = "map.png",
+            Scale =1,
+            Aspect = Aspect.AspectFit,
+            WidthRequest = 25,
+            HeightRequest = 35,
+            HorizontalOptions = LayoutOptions.Center,
+            VerticalOptions = LayoutOptions.Center
+        };
+        stackLayout.Add(image);
+        return new ViewCell { View = stackLayout };
+    });
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> When setting `MapLatLngBounds` and `DistanceType` at the same time, the priority is `Radius` and so calculate zoom level based on radius value.
+
+![MAUI SfMaps zoom level changed image](images/zoom-pan/northSouth_Image.png)
+
+## Double tap zooming
+
+Double tap zooming can be enabled using `EnableDoubleTapZooming` property. The default value of the `EnableDoubleTapZooming` is `false`.
+
+N> It is applicable for both tile layer and shape layer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+            <map:MapTileLayer.ZoomPanBehavior>
+                <map:MapZoomPanBehavior ZoomLevel="2" EnableDoubleTapZooming="True"/>
+            </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 2;
+zoomPanBehavior.EnableDoubleTapZooming = true;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Double tap zooming](images/zoom-pan/double_tap_zoom.gif)
+
+## Animation
+
+### Zoom level Animation
+
+To perform animation on zoom level changing using the `MapTileLayer.EnableZoomingAnimation` property. The default value of the `EnableZoomingAnimation` is `true`.
```

Comment:
To perform the animation on zoom level changing using the `MapTileLayer.EnableZoomingAnimation` property. The default value of the `EnableZoomingAnimation` is `true.`

Reply:
Modified the content